src/variables/utils/astim.ts

```diff
@@ -23,7 +23,10 @@ export const parseASTIM = (query: string): ASTIM => {
   try {
     ast = parse(query)
   } catch (e) {
-    console.error(e)
+    console.error(
+      `ERROR: parseASTIM errored on script:\n ${query}\n with error:`,
```

Comment:
I'd suggest making these two separate log calls, one with the error and one with the query.

Reply:
changed.